avalon/tools/sceneinventory/app.py

```diff
@@ -454,12 +454,13 @@ def get_valid_value(self):
         if text not in lookup:
             return None
 
-        return text
+        return text or None
 
 
 class SwitchAssetDialog(QtWidgets.QDialog):
     """Widget to support asset switching"""
 
+    _fill_check = False
```

Comment:
Shouldn't this be defined in `__init__`?

Reply:
It was there for a reason, but the reason is now irrelevant. So yes, should.